experimental/traffic-portal/src/app/shared/navigation/tp-sidebar/tp-sidebar.component.html

```diff
@@ -13,7 +13,7 @@
 -->
 
 <mat-sidenav-container [autosize]="true">
-	<mat-sidenav #sidenav mode="side" opened>
+	<mat-sidenav #sidenav mode="side" opened [class.hidden]="hidden">
```

Comment:
you don't need to make a CSS class for this, you can just bind to [the HTML `hidden` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden)